flink-table-store-format/src/main/java/org/apache/flink/table/store/format/orc/OrcFileFormat.java

```diff
@@ -97,8 +98,8 @@ public BulkWriter.Factory<RowData> createWriterFactory(RowType type) {
 
         return new OrcBulkWriterFactory<>(
```

Comment:
There is conf in `OrcBulkWriterFactory` too.
Maybe we can create a new `OrcBulkWriterFactory` in table store to avoid new configuration.

Reply:
@JingsongLi I have copied OrcBulkWriterFactory from flink and updated the code to reuse writer config